evap/staff/importers.py

```diff
@@ -182,6 +244,7 @@ def create(self, name_de, name_en, degree_names, course_type_name, is_graded, re
             is_graded=is_graded,
             responsible_email=responsible_email,
             errors=errors,
+            existing_course=None,
```

Comment:
Note: We could make this a default on the dataclass itself by specifying `= None` on the field declaration, but it would also be a bit confusing, so let's keep it like it is right now :)